package.json

```diff
@@ -21,7 +21,7 @@
     "start": "craco start",
     "build": "craco build",
     "serve": "serve build -l 3000",
-    "lint": "yarn eslint --ignore-path .gitignore .",
+    "lint": "yarn eslint --ignore-path .gitignore --cache --cache-location node_modules/.cache/eslint/ .",
```

Comment:
This should be changed to match the lint location set by react-scripts. I will make that explicit in craco.config.js so that maintenance is simpler.